test/Handler/CommentsSpec.hs

```diff
@@ -11,30 +17,32 @@ spec :: Spec
 spec = withApp $ do
     describe "GET CommentsR" $ do
         it "returns a list of comments by article" $ do
-            Entity siteId _ <- createSite
-
-            u <- createUser "1"
-            c1 <- createComment (entityKey u) siteId "1" "1" "1"
-            c2 <- createComment (entityKey u) siteId "1" "2" "2"
-            c3 <- createComment (entityKey u) siteId "2" "1" "3"
+            siteId <- runDB $ insert $ buildSite
+            user <- runDB $ createUser "1"
+            (c1, c2, c3) <- runDB $ (,,)
```

Comment:
> Is the desire to avoid repeating runDB?

Technically, less `runDB` calls should perform better. Aesthetically, the additional `runDB`s often cause the line to go over 80 and require awkward breaking (this may explain my `user`/`u` waffle as well). I'll still remove the Applicative, but that was the true reasoning.